content/guides/building/pre-reqs.md

```diff
@@ -53,18 +89,18 @@ Note that in addition to all the platform-specific packages, you will also need
 | [CentOS](http://centos.org)        | [rpm/yum](#yum)      | YES       |                     |
 | [Debian](http://debian.org)        | [deb/apt](#apt)      | YES       |                     |
 | [Fedora](https://fedoraproject.org)| [rpm/dnf](#yum)      | YES       |                     |
-| [FreeBSD](http://freebsd.org)      | [packages](#bsd)     | YES       |                     |
+| [FreeBSD](http://freebsd.org)      | [packages](#bsd)     | YES       | Strange building issues have been reported before |
```

Comment:
Is there a bugreport? Or is that only hearsay?

In any case, putting something like "Not frequently tested" sounds better than "Strange issues"

Reply:
True.